src/Stratis.Bitcoin.Features.SmartContracts/ReflectionExecutor/Controllers/SmartContractsController.cs

```diff
@@ -113,7 +129,15 @@ public IActionResult GetBalance([FromQuery]string address)
             return Json(balance);
         }
 
-        [Route("storage")]
+        /// <summary>
+        /// Gets a single piece of stored smart contract data. The data is held in the SmartContract.PersistedData property,
```

Comment:
`Performs a lookup in the smart contract state database for the supplied contract address and storage key. Returns the value associated with the given key, deserialized as the specified data type. If the key does not exist or deserialization fails, will return a default value for that type.`